ddtrace/llmobs/_llmobs.py

```diff
@@ -267,6 +281,8 @@ def export_span(cls, span: Optional[Span] = None) -> Optional[ExportedLLMObsSpan
         if span.span_type != SpanTypes.LLM:
             log.warning("Span must be an LLMObs-generated span.")
             return None
+
+        record_export_span_usage(config._llmobs_ml_app)
```

Comment:
Do we need to call `record_export_span_usage()` twice in this function? Can we just call it once at the top of the function for simplicity?

Reply:
i did it so that we capture only the successful calls. as i suggested in the PR description, I'd recommend to have a separate PR in which we can capture errors of interest (either by relaying logs or introducing failure metrics)